### PR TITLE
fix(esx-legacy): Updated/added es translations & changed the default sql files from fr to en

### PR DESCRIPTION
**Pull request by paarbo**

### CODE REVIEW

**Comment by Gellipapa** [comment]
Hi! Thank you for help @paarbo but check and delete the unnecessary spaces thanks. Once this is done I merge pr.

**Review by TheFantomas — CHANGES_REQUESTED** [review]
Please update all es.lua to newest version. Like en.lua :D Because a few things have already been removed a long time ago :D For example like this
image

**Review by TheFantomas — APPROVED** [review]

**Comment by Gellipapa** [comment]
> Also I have included in this request, some improvements to the server.cfg. (These changes have been also pushed to the esx-recipe)

Hi! @paarbo! Okey thanks for completing the server cfg. It is being reviewed and will be available soon.

File: [esx_addons]/esx_ambulancejob/esx_ambulancejob.sql [hunk 1]
```diff
@@ -1,24 +1,24 @@
 INSERT INTO `addon_account` (name, label, shared) VALUES
```
Comment (Gellipapa) [inline [esx_addons]/esx_ambulancejob/esx_ambulancejob.sql:1]: Default SQL please do not edit! Replace with the original.

File: [esx_addons]/esx_ambulancejob/localization/en_esx_ambulancejob.sql [hunk 2]
```diff
@@ -1,33 +0,0 @@
-
```
Comment (Gellipapa) [inline [esx_addons]/esx_ambulancejob/localization/en_esx_ambulancejob.sql:1]: Please don't remove EN version!

File: [esx_addons]/esx_bankerjob/localization/en_esx_bankerjob.sql [hunk 3]
```diff
@@ -1,18 +0,0 @@
-
```
Comment (Gellipapa) [inline [esx_addons]/esx_bankerjob/localization/en_esx_bankerjob.sql:1]: Please don't remove EN version!

File: [esx_addons]/esx_animations/localization/en_config.lua [hunk 4]
```diff
@@ -1,162 +0,0 @@
-Config = {}
```
Comment (Gellipapa) [inline [esx_addons]/esx_animations/localization/en_config.lua:1]: Please don't remove EN version!

**Review by Gellipapa — COMMENTED** [review]
Please fixes all review comments.

**Comment by TheFantomas** [comment]
Why dont remove en version? He move en version to "default sql version" and move default sql to france :D Because default sql is  in french :D

**Comment by Gellipapa** [comment]
> 

I know it's just distracting, I'll delete it :)

**Comment by Gellipapa** [comment]
> > > 
> > 
> > 
> > I know it's just distracting, I'll delete it :)
> 
> All sql's files outside of localization folder will be deleted? @Gellipapa

The point is that the default SQL should not be in the locales folder and should be the default script name. As is the default when you look at a script.

**Comment by Gellipapa** [comment]
> > > > > 
> > > > 
> > > > 
> > > > I know it's just distracting, I'll delete it :)
> > > 
> > > 
> > > All sql's files outside of localization folder will be deleted? @Gellipapa
> > 
> > 
> > The point is that the default SQL should not be in the locales folder and should be the default script name. As is the default when you look at a script.
> 
> Then I don't see the point of restoring the deleted sql files that you asked

If you can solve it the way it was in the original, there's no question of how to solve it.

**Pull request closed without merging**